pkg/controllers/vdb/http_server.go

```diff
@@ -20,12 +20,16 @@ import (
 
 	vapi "github.com/vertica/vertica-kubernetes/api/v1beta1"
 	"github.com/vertica/vertica-kubernetes/pkg/events"
+	vmeta "github.com/vertica/vertica-kubernetes/pkg/meta"
 	corev1 "k8s.io/api/core/v1"
 )
 
 // hasCompatibleVersionForHTTPServer checks, in case http server is enabled, if
 // the server has a required version for http server.
 func hasCompatibleVersionForHTTPServer(vrec *VerticaDBReconciler, vdb *vapi.VerticaDB, logEvent bool, action string) bool {
+	if vmeta.UseVClusterOps(vdb.Annotations) {
```

Comment:
I assume this is needed because we need the http service to communicate with the library. Did the vertica version check not work?

Reply:
I might have overthought this one. I thought that given that we use the feature only for development we did not care about all the checks, we would assume http server is running. I also did not want it to depend on `httpServerMode`.We don't need this. I suppose by the time vclusterOps is made public, http server will be mandatory and `httpServerMode` may be deprecated.

Reply:
Yeah, we'll probably need to have smarter checks in the operator that if you are using the version of vertica without admintools that you also need the http service started. Lets leave this off for now.